content/2022/videookulografija.md

```diff
@@ -1,4 +1,561 @@
+Potom, traže se prečnik i centar kruga. 
+Više radova se zasniva na binarizaciji slike implementirane korišćenjem različitih metoda za određivanje paraga. 
+Neke od njih su - globalni ili lokalni adaptivni prag. 
+Od 6 korišćenih PDA algoritama, većina koristi novu metodu za binarizaciju slike, implementiranu na adaptivnoj kvantitativnoj segmentaciji. 
+U referentnim radovima postoje primeri uklanjanja šumova tehnikom bazirane na maksimalno određenom prostoru u praćenju rekonstrukcije zenice. 
+Svi radovi uključuju Hafovu transformaciju kruga i morfološke operacije (erozija i dilatacija). 
+Rad [5] za detekciju ivica kruga koristi derivat drugog reda rezultujuće slike, što je postignuto Laplacom od Gausovog filtera. 
+Sledeći korak je određivanje centra zenice algoritnom centroidne metode čime je uspešno detektovana zenica.
+
+
+
+## **3. Aparatura i metode**
+
+
+
+Ovaj rad se sastoji iz dva dela - aparture pomoću koje se vrši akvizicija slike i metoda kojima je postignuta obrada slike i kalibracija.
+
+
+
+#### **3.1 Aparatura**
+
+
+
+Na slici 1 i 2 može se videti korišćena aparatura sastavljena iz web kamera, držača za web kameru i naočara koji su 3D odštampane.
+Web kamera je korišćena za praćenje zenice sa koje je skinut IR filter kako bi detektovanje zence bilo olakšano za sve boje dužica. 
+3D model držača ima dve značajne opcije za podešavanje:
+
+1. Pomeranjem šarke na koju je pričvršćena web kamera, može se približiti ili udaljiti od oka u odnosu na potrebe korisnika
+
+
+2. Web kamera se može pomerati po y osi uz pomoć klizača za podešavanje visine web kamere
+
+
+Web kamera ja pričvršćena na naočare gde su postavljene i led diode kako bi osvetljenje slike bilo konstantno.
+
+![Slika hardvera](/images/2022/videookulografija/OPREMA1.jpg)                    ![Slika hardvera](/images/2022/videookulografija/OPREMA2.jpg)
+
+
+(slika 1 - Izgled naučara)               (slika 2 - Izgled naočara)       
+
+
+
+#### **3.2. Metode**
+
+
+Metode se sastoje iz dve faze:
+1. Obrada slike, ondnosno frejmova snimka u realnom vremenu;
+2. Obrade podataka, odnosno filtriranje podataka i kalibracija.
+
+
+##### **3.2.1. Obrada slike**
+
+
+Kako bi zenica bila detektovana neophodno je pronaći koordinate centra zenice. 
+Obrađivanjem slike, tačnije svakog frejma snimka u realnom vremenu, dobijaju se podaci o koordinatama centra zenice.
+Proces obrade slike se sastoji iz sledećih koraka:
+
+1. Prebacivanje slike iz RGB u grayscale prostor boja
+2. Zamućivanje slike
+3. Binarizacija slike
+4. Morfološke operacije
+5. Detekcija ivica pomoću Keni algoritma
+6. Detekcija krugova, odnosno zenice na slici pomoću Hafove transformacije
+
+
+#### 3.2.1.1. Prebacivanje slike iz RGB u grayscale prostor boja
+
+Slika u RGB spektru ima isuviše informacija koje nam nisu potrebne, dok slike u grayscale prostoru sadrže sve informacije koje su nam potrebne.
```

Comment:
ima više informacija nego što nam je potrebno, a takođe je kompleksnija za obradu, pa zato biramo grayscale